ui/helpers/constants/transactions.js

```diff
@@ -19,3 +19,8 @@ export const TOKEN_CATEGORY_HASH = {
   [TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER]: true,
   [TRANSACTION_TYPES.TOKEN_METHOD_TRANSFER_FROM]: true,
 };
+
+export const TRANSACTION_ENVELOPE_TYPE_NAMES = {
```

Comment:
Is it worth it, do you think, to put this in `shared/constants/transaction.js` after `TRANSACTION_ENVELOPE_TYPES`, since they're related to each other?

Reply:
The code would then be minified into both the background and the client side; since it's not used in the background, this seems like the right place.

Reply:
Ah, yeah, that's a good point 👍🏻